test/Service/Lti/EncryptFactoryLtiAuthAdapterServiceTest.php

```diff
@@ -41,16 +43,39 @@ public function testCreate()
      */
     protected function mockService()
     {
-        $adapter = $this->getMockBuilder(LtiAuthAdapter::class)->disableOriginalConstructor()->getMock();
-
         $service = $this->getMockBuilder(EncryptFactoryLtiAuthAdapterService::class)->disableOriginalConstructor()
-            ->setMethods(['callParentCreate', 'propagate'])
+            ->setMethods(['propagate'])
             ->getMockForAbstractClass();
 
-        $service
-            ->method('callParentCreate')
-            ->willReturn($adapter);
-
+        $service->setServiceLocator($this->mockServiceLocator());
         return $service;
     }
+
+    protected function mockServiceLocator()
```

Comment:
You can use TestCase::getServiceLocatorMock() from generis extension, to do not duplicate the same logic. It accepts array in format [$serviceId => $serviceMock]